app/controllers/katello/api/v2/alternate_content_sources_bulk_actions_controller.rb

```diff
@@ -34,6 +34,21 @@ def refresh_alternate_content_sources
       end
     end
 
+    api :POST, '/alternate_content_sources/bulk/refresh_all', N_("Refresh all alternate content sources")
+    def refresh_all_alternate_content_sources
+      refreshable_alternate_content_sources = AlternateContentSource.readable.editable
```

Comment:
I think editable should be enough for permissions here. Editable but not readable doesn't make much sense..I don't know how that would work..

Reply:
Added in your suggested changes, thank you.